File: 프로그래머스/lv1/12930. 이상한 문자 만들기/이상한 문자 만들기.js
function solution(s) {
    const arr = s.split(' ');
    const answer = [];
    for(let i = 0; i < arr.length; i++){
        const tempArr = [];
        for(let j = 0; j < arr[i].length; j++){
            const character = arr[i].charAt(j)
            if(j % 2 === 0){
                tempArr.push(character.toUpperCase());
            } else {
                tempArr.push(character.toLowerCase());
            }
        }
        answer.push(tempArr.join(''));
    }

    return answer.join(' ');
}

File: 프로그래머스/lv1/12930. 이상한 문자 만들기/이상한 문자 만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        String[] strArr = s.split(" ", -1);
        String[] tempArr = new String[strArr.length];
        
        for(int i=0; i < strArr.length; i++){
            String tempStr = "";
            for(int j=0; j < strArr[i].length(); j++){
                if(j % 2 == 0){
                    tempStr += Character.toUpperCase(strArr[i].charAt(j));
                } else {
                    tempStr += Character.toLowerCase(strArr[i].charAt(j));
                }
            }
            tempArr[i] = tempStr;
        }
        return String.join(" ", tempArr);
}
}